tests/test_wellcompletiondata.py

```diff
@@ -89,6 +89,19 @@ def test_df2pyarrow():
     pd.testing.assert_frame_equal(df, _df2pyarrow(df).to_pandas(), check_like=True)
 
 
+def test_metadata():
+    """Test that the KH column has metadata and that unit is mDm"""
+    eclfiles = EclFiles(EIGHTCELLS)
+    df = wellcompletiondata.df(
+        eclfiles, zonemap=EIGHTCELLS_ZONEMAP, use_wellconnstatus=False
+    )
+    table = _df2pyarrow(df)
```

Comment:
Test the dataframe as well for metadata, not only how it gets processed to pyarrow?

Reply:
Am I not doing that in the previous test: `test_df2arrow`? Should I merge the two tests maybe?

Reply:
Oh, you mean the metadata in the dataframe. Yes I see, I will do that.